CHANGELOG.md

```diff
@@ -21,6 +21,8 @@
   checked by the writing storage imports `db_write` and `db_remove`. Read-only
   mode is off by default for backwards compatibility. `call_query_raw` now sets
   the instance's storage to readonly.
+- A new error case `VmError::WriteAccessDenied` was created that is returned
+  when a contract tries to call an import that potentially writes to storage.
```

Comment:
I would add "during a query" or something. This makes it sound like it should never write to storage.